content/pages/greenhouse.md

```diff
@@ -4,6 +4,8 @@ author: Anders Nord
 type: index
 status: published
 
+<img src="data/poster2.jpeg" width=100%>
```

Comment:
Then you have to add a copy to the build action too.

Reply:
I was doing that, but you were faster than me :) Thanks again!